packages/wrangler/src/dev.tsx

```diff
@@ -461,25 +461,34 @@ function useEsbuild(props: {
             else {
               // nothing really changes here, so let's increment the id
               // to change the return object's identity
-              setBundle((previousBundle) => ({
-                ...previousBundle,
-                id: previousBundle.id + 1,
-              }));
+              setBundle(
+                (previousBundle) =>
+                  previousBundle && {
```

Comment:
I would rather assert that it is defined, and throw if not. (In practice it will never be undefined, because this will be called only after the first build) Alternately, a non-null assertion?

Reply:
I'll add an error.
But I don't actually understand what the point of this handler is. Is the idea that updating `bundle` by calling `setBundle()` causes the `Dev` component to be re-rendered?